lms/templates/courseware/course_navigation.html

```diff
@@ -39,6 +42,10 @@
 % if disable_tabs is UNDEFINED or not disable_tabs:
     <%
     tab_list = get_course_tab_list(request.user, course)
+
+    # Record course goals user activity for (web) courseware and course tabs that are outside of the learning mfe
+    if RECORD_USER_ACTIVITY_FLAG.is_enabled():
+        UserActivity.record_user_activity(user, course.id)
```

Comment:
Oy, this is better than being in `get_course_tab_list` - but another weird little place we're squirreling these away.

Reply:
open to other locations if you have suggestions